src/resources/triton_python_backend_utils.py

```diff
@@ -310,7 +310,7 @@ def __init__(self, model_config):
         self._model_config = json.loads(model_config)
 
     def __str__(self):
```

Comment:
Can we remove this function?

Reply:
When returning `auto_complete_model_config`, this function will be called so that the returned `auto_complete_model_config` will have a converted json format. So I added `json.dumps` here and removed `FixStringToJsonFormat` function.
```
def __str__(self):
        return json.dumps(self._model_config)
```